Dockerfile

```diff
@@ -1,12 +1,13 @@
-FROM node:11
+FROM node:13.8.0-alpine3.11
```

Comment:
NodeJS version is updated to 13.8.0 but Travis is building with 8.x.x [ [.travis.yml](https://github.com/asyncapi/generator/blob/41efaf8bce4098702995f4b58c55ef442d1ee593/.travis.yml#L3) ]. Would suggest bumping the build-version up to 13.8.0 as well.

Reply:
Not sure if it is a big problem if the previous version was 11, so still not the same as the one used in Travis. I suggest to leave it as it is and move travis to github actions later.

Reply:
@derberg Alright I can live with that. As long as it's on the radar 👍 .